hardware/hardware.mk

```diff
@@ -1,6 +1,11 @@
 CPU_SRC_DIR:=$(CPU_DIR)/hardware/src
 VSRC+=$(CPU_SRC_DIR)/picorv32.v $(CPU_SRC_DIR)/iob_picorv32.v
 
+#use hard multiplier and divider instructions
+ifeq ($(USE_MUL_DIV),1)
```

Comment:
separate in 2 macros for mul and div

Reply:
The instructions are enabled for both, that's why I did like this.